docs/user/security/authorization/index.asciidoc

```diff
@@ -6,7 +6,21 @@ The Elastic Stack comes with the `kibana_admin` {ref}/built-in-roles.html[built-
 
 When you assign a user multiple roles, the user receives a union of the roles’ privileges. Therefore, assigning the `kibana_admin` role in addition to a custom role that grants {kib} privileges is ineffective because `kibana_admin` has access to all the features in all spaces.
 
-NOTE: When running multiple tenants of {kib} by changing the `kibana.index` in your `kibana.yml`, you cannot use `kibana_admin` to grant access. You must create custom roles that authorize the user for that specific tenant. Although multi-tenant installations are supported, the recommended approach to securing access to {kib} segments is to grant users access to specific spaces.
+[[xpack-security-multiple-tenants]]
+==== Supporting multiple tenants
+
+There are two approaches to supporting multi-tenancy in {kib}:
+
+1. *Recommended:* create a space and a limited role for each tenant, and configure each user with the appropriate role.
+2. deprecated:[7.13.0,"In 8.0 and later, the `kibana.index` setting will no longer be supported."] Set up separate {kib} instances to work
+with a single {es} cluster by changing the `kibana.index` setting in your `kibana.yml` file.
++
+NOTE: When using multiple {kib} instances this way, you cannot use the `kibana_admin` role to grant access. You must create custom roles
+that authorize the user for each specific instance.
+
+Whichever approach you use, be aware that you need to be careful when granting cluster privileges and index privileges. Both of these
+approaches share the same {es} cluster, and {kib} spaces do not prevent you from granting users of two different tenants access to the same
+index.
```

Comment:
Kibana's feature privileges alone won't grant direct access to any indices. This was the clearest and most concise way I could think of to address @grabowskit's concerns in https://github.com/elastic/kibana/issues/96101#issuecomment-818990813, but if anyone has suggestions on how this could be worded differently, please do share!